app/config/ezplatform.yml

```diff
@@ -37,5 +37,13 @@ ezpublish:
                 full:
                     home:
                         template: full\home.html.twig
+                        controller: AppBundle:Site:show
```

Comment:
IMO this should be called `AppBundle:Home:show` because it's dedicated to the Home view.